modules/transactions/src/main/java/org/apache/ignite/internal/tx/impl/VolatileTxStateMetaStorage.java

```diff
@@ -171,39 +170,34 @@ public CompletableFuture<Void> vacuum(
                 } else if (TxState.isFinalState(meta0.txState())) {
                     Long initialVacuumObservationTimestamp = meta0.initialVacuumObservationTimestamp();
 
-                    if (initialVacuumObservationTimestamp == null && txnResourceTtl > 0) {
-                        markedAsInitiallyDetectedTxnsCount.incrementAndGet();
```

Comment:
`markedAsInitiallyDetectedTxnsCount` is it still needed?

Reply:
Done.